mismi-s3/mismi-s3.cabal

```diff
@@ -80,7 +80,7 @@ executable s3
                      , ambiata-x-optparse
                      , ambiata-x-eithert
                      , filepath
-                     , optparse-applicative            == 0.11.*
+                     , optparse-applicative            == 0.12.*
```

Comment:
I'm still not sure about this bit - got a funny feeling it might force an upgrade in other places, but @markhibberd might know for sure